src/api/v1beta1/properties.go

```diff
@@ -166,6 +166,10 @@ func (dk *DynaKube) Image() string {
 		return dk.Spec.OneAgent.ClassicFullStack.Image
 	} else if dk.HostMonitoringMode() {
 		return dk.Spec.OneAgent.HostMonitoring.Image
+	} else if dk.CloudNativeFullstackMode() {
+		return dk.Spec.OneAgent.CloudNativeFullStack.CodeModulesImage
```

Comment:
shouldn't these be in there own func ?  `func (dk *DynaKube) CodeModulesImage() string {...}`
there will be an image field for cloudNative that will use this func

Reply:
you are right, did not think about that. thank you.